sources/ca/on/dufferin.json

```diff
@@ -10,6 +10,14 @@
             {
                 "name": "county",
                 "data": "https://services5.arcgis.com/C0BEdgagVaXzcw0W/arcgis/rest/services/Dufferin_County_Entrances/FeatureServer/0",
+                "website": "https://data-dufferincounty.opendata.arcgis.com/datasets/address-points-dufferincounty-open-data",
+                "license": {
+                    "url": "https://www.arcgis.com/sharing/rest/content/items/fbbfe060de804d44a4e39cd51a451e35/data",
```

Comment:
This is the URL to the data listing, not to the license. Can you find a link that's specific to the license?

Reply:
https://www.arcgis.com/sharing/rest/content/items/fbbfe060de804d44a4e39cd51a451e35/data takes me to The County of Dufferin open government license PDF

Reply:
Ah yep, I must have been looking at the wrong tab.